src/test/java/uk/gov/justice/hmpps/prison/executablespecification/BookingStepDefinitions.java

```diff
@@ -499,4 +501,10 @@ private LocalDate convertTodayAndParse(final String dateCreated) {
         if (dateCreated.equals("today")) return LocalDate.now();
         return LocalDate.parse(dateCreated);
     }
+
+    private LocalDateTime convertTodayAndParseDateTime(final String dateTimeCreated) {
```

Comment:
slightly odd field name here since it isn't always a creation date time.  Agree that you've copied the code above, but I'd prefer just a `dateTime` or even better `dateTimeAsString`